airbyte-integrations/connectors/source-plausible/source_plausible/schemas/stats.json

```diff
@@ -34,4 +34,4 @@
       "description": "The number of visits/sessions."
     }
   }
-}
```

Comment:
Please remove the file from this PR as it is unrelated to the LinkedIn Ads source connector.